src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -31,8 +31,8 @@
     /* Sample tiny config file:
        <ncwms>
          <wfsServer>http://localhost:8080/geoserver/ows</wfsServer>
-         <urlSubstitution key="/mnt/imos-t3/IMOS/opendap/">http://thredds-1-aws-syd.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
-         <urlSubstitution key="^/IMOS/">http://thredds-1-aws-syd.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
+         <urlSubstitution key="/mnt/imos-t3/IMOS/opendap/">http://thredds.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
```

Comment:
Not sure we should be hard coding our infrastructure urls into the code.

All the other geoserver extensions at least try to do this stuff properly and push out this configuration into serialized structures, with a GUI over the top.